components/ProductCard/ProductCard.tsx

```diff
@@ -13,9 +15,18 @@ interface CardProps {
 const Card: React.FC<CardProps> = ({ product, path }) => {
   let pathname = usePathname()
   let { itemId, name, fullPrice, price, screen, capacity, ram, image } = product;
+  const [faovrite, setFavotire] = useState<boolean>(false);
   const storage = useLocalStorage('cart');
+  const favorites = useLocalStorage('favorites');
```

Comment:
Zróbmy jakiś obiekt z takimi stałymi (może być np. `/constants/localStorageKeys`) i używajmy ich wszędzie, zamiast stosować stringi:
```javascript
const LOCAL_STORAGE_KEYS = {
  FAVORITES: "FAVORITES",
  // other keys...
}
```

To samo podejście można zastosować do nazwa kolekcji z bazy danych